public/index.ts

```diff
@@ -927,13 +927,16 @@ function onVolumeCreated(volume: Volume, isTimeSeries = false, frameNumber = 0)
       // create the main volume and add to view (this is the only place)
       myState.volume = new Volume(myJson);
 
+      const atlasWidth = myJson.tile_width * myJson.cols;
```

Comment:
This maybe a holdover from an old design choice, but is there a reason for the image info to be named `myJson` here?

Reply:
Yes - this viewer used to only load from one single particular file format, and it was represented as a json file that mapped directly to the ImageInfo data structure.